worker/create_batches_worker_test.go

```diff
@@ -46,6 +46,8 @@ var _ = Describe("CreateBatches Worker", func() {
 	w := worker.NewWorker(logger, GetConfPath())
 	createBatchesWorker := worker.NewCreateBatchesWorker(w)
 	createCSVSplitWorker := worker.NewCSVSplitWorker(w)
+	processBatchWorker := worker.NewProcessBatchWorker(w)
+	processBatchWorkerConcurrency := w.Config.GetInt("workers.processBatch.concurrency")
```

Comment:
Is this config new? What exactly does it stand for?

Reply:
The config is old, but I have to set it on this test because we need to start the processBatchWorker. The idea is to use the middleware to evaluate the totalUsers and totalTokens at the right moment that the processBatchWorker starts